### ISSUE TITLE
The code that you submitted to the Fishyscapes submission?

### ISSUE THREAD

**Issue by pswena** [giandbt/synboost#15]
I saw the issue 13, but could you please upload your code which you submitted to the Fishyscapes submission to get the result of table 1? Please.

**Comment by hermannsblum** [comment]
Sure, you can find that code here: https://github.com/hermannsblum/fishyscapes/pull/4

**Comment by pswena** [comment]
> Sure, you can find that code here: [hermannsblum/fishyscapes#4](https://github.com/hermannsblum/fishyscapes/pull/4)

Thank you very much! Thank you very much! Cause I am a freshman in deep-learning, so I asked lots of questions. Thank you for you reply.

**Comment by pswena** [comment]
> 

Firstly, I can't find test_fishy_torch in driving_uncertainty. Is that you missing this file?
Secondly, I want to replicate the result of table 1, should I submit the code you mentioned before to the Fishyscapes submission?

**Comment by hermannsblum** [comment]
I may miss the point, but you cannot replicate the result of table 1 yourself. The results for Synboost are available at https://fishyscapes.com/results . Table 1 is literally copy-pasted from there.

**Comment by pswena** [comment]
> 

I just want to replicate your results of table 1 in FS Lost & Found 、FS Static and FS Web datasets. So what should I do? Which code should I submit to the Fishyscapes submission?  Or, I cannot replicate by myself.

**Comment by hermannsblum** [comment]
No, you cannot replicate this by yourself. The idea of public benchmarks and challenges is that their test-sets are hidden and only they can measure/evaluate your models, such that you cannot overfit on the test sets.

**Comment by hermannsblum** [comment]
If you want to try it out on some additional images, I would recommend these: https://segmentmeifyoucan.com/ The test-images in that benchmark are available for download, so you can run Synboost over them.

**Comment by pswena** [comment]
> If you want to try it out on some additional images, I would recommend these: https://segmentmeifyoucan.com/ The test-images in that benchmark are available for download, so you can run Synboost over them.

Thanks for your quick responses and congratulations for the paper! Thank you very much.

**Comment by pswena** [comment]
> I may miss the point, but you cannot replicate the result of table 1 yourself. The results for Synboost are available at https://fishyscapes.com/results . Table 1 is literally copy-pasted from there.

May I ask you for the test_fishy_torch in driving_uncertainty?
